.ci/containers/build-environment/Dockerfile

```diff
@@ -39,8 +39,8 @@ RUN git config --global user.email "magic-modules@google.com"
 RUN go install golang.org/x/tools/cmd/goimports@d088b475e3360caabc032aaee1dc66351d4e729a
 RUN go install github.com/github/hub@v2.11.2+incompatible
 
-ADD "https://raw.githubusercontent.com/GoogleCloudPlatform/magic-modules/main/mmv1/Gemfile" Gemfile
-ADD "https://raw.githubusercontent.com/GoogleCloudPlatform/magic-modules/main/mmv1/Gemfile.lock" Gemfile.lock
+ADD "https://raw.githubusercontent.com/GoogleCloudPlatform/magic-modules/refs/heads/legacy-ruby/mmv1/Gemfile" Gemfile
```

Comment:
We can probably remove ruby & the gemfiles from the build environment now?

Reply:
Correct, but breaking it up into multiple steps (the part 1 is actually updating main to use these legacy references, and this part 2 was layered on that)